### ISSUE TITLE
Not working properly with TexLive 2021 and 2022

### ISSUE THREAD

**Issue by HMegh** [jkuczm/mmacells#52]
Hi, 

I have been using mmacells for a while on Overleaf. However, I have to change the TexLive version to 2020 in order for it to work. 

When using TexLive 2021, I get the following error:

```
! Undefined control sequence.
<argument> ...otalleftmargin \hbox_unpack_clear:N 
                                                  \@labels 
l.143 \begin{mmaCell}{Code}
                           
The control sequence at the end of the top line
of your error message was never \def'ed. If you have
misspelled it (e.g., `\hobx'), type `I' and the correct
spelling (e.g., `I\hbox'). Otherwise just continue,
and I'll forget about whatever was undefined.
```

When using the 2022 version, there are no errors but the In[x] and Out[x] do not appear: Here's what this code prints

```
\begin{mmaCell}[]{Code}
    Apart[1/(s^2-1)]
\end{mmaCell}
\begin{mmaCell}{Output}
    1/(2 (-1 + s)) - 1/(2 (1 + s))
\end{mmaCell}
```

<img width="495" alt="Screen Shot 2022-08-25 at 1 47 38 PM" src="https://user-images.githubusercontent.com/8241188/186734663-21fbb597-1d86-4122-a5a6-6c064b3b00e3.png">

**Comment by jkuczm** [comment]
This should be fixed by #42, which is already merged, but not released yet. Newest [mmacells.sty file from master branch](https://raw.githubusercontent.com/jkuczm/mmacells/master/mmacells.sty) should work properly.

**Comment by HMegh** [comment]
Yes, that fixed it. Thanks!!
